ubyssey/static/src/styles/modules/widgets/_upcoming_events_horizontal.scss

```diff
@@ -0,0 +1,55 @@
+.c-widget-upcoming-events-horizontal {
+  background-color: #f5f5f5;
+  margin: 15px 0;
+  padding: 25px 50px;
+  box-sizing: border-box;
+  border-top: solid 1px $color-border-gray;
+  border-bottom: solid 1px $color-border-gray;
+
+  @media($bp-smaller-than-tablet) {
+    padding: 10px 0;
+  }
+
+  text-transform: uppercase;
+}
+
+.c-widget-upcoming-events-horizontal__label-row {;
+  display: flex;
+
+  // Text
+  color: $color-text-gray;
+  font-size: font-size(14);
+  font-weight: $font-weight-semi-bold;
+  letter-spacing: 0.045em;
+}
+
+.c-widget-upcoming-events-horizontal__label {
+  flex: 1;
+}
+
+.c-widget-upcoming-events-horizontal__view-all {
+  text-align: right;
+}
+
+.c-widget-upcoming-events-horizontal__events-row {
+  margin-top: 0.75rem;
+  display: flex;
+  justify-content: space-between;
+
+  @media($bp-smaller-than-tablet) {
+    display: block;
+
+    .c-event:not(:last-child) {
+      border-bottom: 0;
+    }
+  }
+
+}
+
+.c-widget-upcoming-events-horizontal__event-wrapper {
+  flex-basis: 32%;
+
+  .c-event {
+    height: 100%;
```

Comment:
I had to add height: 100% here to make them the same height within the flex grid